code/modules/xenobio2/mob/xeno procs.dm

```diff
@@ -64,6 +64,10 @@ Procs for targeting
 
 			reagent_response = chemreact[R.id]
 
+			if(!reagent_response)
+				temp_chem_holder.reagents.clear_reagents()
+				return 1
```

Comment:
I checked whether chems that actually had a function worked properly or not, and it appears that they did, reagent_response will only be null if the chem doesn't do anything to slimes.